src/core/ext/transport/chttp2/client/secure/secure_channel_create.c

```diff
@@ -97,15 +98,15 @@ grpc_channel *grpc_secure_channel_create(grpc_channel_credentials *creds,
   GRPC_API_TRACE(
       "grpc_secure_channel_create(creds=%p, target=%s, args=%p, "
       "reserved=%p)",
-      4, (creds, target, args, reserved));
+      4, ((void *)creds, target, (void *)args, (void *)reserved));
   GPR_ASSERT(reserved == NULL);
   // Make sure security connector does not already exist in args.
```

Comment:
(what sort of chronological ordering does github use for comment threads, seriously...)

I'm partial to Julian's second suggestion involving `check_peer`. That being said, I'm in the minority :) Given we have options to go back to if performance issues arise, I'll go ahead and reintroduce the creation of the security connector for every subchannel.